lib/oli_web/live/delivery/student/utils.ex

```diff
@@ -219,7 +219,7 @@ defmodule OliWeb.Delivery.Student.Utils do
     </div>
     <div
       id={"objective_#{@objective.resource_id}_tooltip"}
-      class="hidden absolute h-[57px] px-6 pt-[15px] pb-6 -top-[20px] -left-6 text-gray-800 dark:text-gray-700 text-base font-normal font-['Inter'] leading-normal bg-gray-300 dark:bg-white rounded-md border-2 border-gray-700 flex-col justify-start items-start gap-4 -translate-x-full"
+      class="hidden absolute min-h-[57px] max-w-[240px] -top-[20px] p-6 -left-6 text-gray-800 dark:text-gray-700 text-base font-normal font-['Inter'] leading-normal bg-gray-300 dark:bg-white rounded-md border-2 border-gray-700 flex-col justify-start items-center gap-4 z-10 -translate-x-full"
```

Comment:
We could use tailwind default values instead of custom here:
`max-w-60`
`min-h-14` (it is 56 instead of 57 pixels but it should work)